<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferWindow</class>
 <widget class="QWidget" name="TransferWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widgetToolBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="widgetToolbar_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignRight">
    <widget class="ThemedButton" name="buttonClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" alignment="Qt::AlignLeft">
    <widget class="ThemedButton" name="buttonClear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Clear List</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="lmcTransferListView" name="listWidgetTransferList">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelSelectedUser">
     <property name="text">
      <string>Show downloads from user:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ThemedComboBox" name="comboBoxSelectedUser">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="bottomLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ThemedButton</class>
   <extends>QToolButton</extends>
   <header location="global">themedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ThemedComboBox</class>
   <extends>QComboBox</extends>
   <header>themedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>lmcTransferListView</class>
   <extends>QListView</extends>
   <header>transferlistview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listWidgetTransferList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
